tests/Unit/OperatorTest.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Tests\Unit;
+
+use App\Models\Operator;
+use Tests\TestCase;
+
+class OperatorTest extends TestCase
+{
+    public function testGetGridAreaForOperator() {
```

Comment:
Jeg ved godt det ikke er relevant for det her PR.. men jeg syntes at PHPStan bør inkludere test filer .. og at vi måske skal overveje at installere Laravel Pint ( Det er bare PHP CS Fixer i laravel forklædning ,, men det er helt nyt og kunne være fedt at prøve af  )

Link: https://laravel.com/docs/9.x/pint

Reply:
@Tvup Faktisk ved jeg ikke om jeg er så stor fan af Pint alligevel. Det ser ud til den kun kan køre ved at fixe det hele for en .. det er jeg ikke fan af. Jeg ville gerne have en del af pipelinen, bare lavede tjekket uden at køre fix.

Så det kan være jeg laver et PR - med native PHP_CS_FIXER istedet for 😅

Reply:
Ja, det har jeg jo ikke prøvet før, men hvis PHPStan også kan gøre en bedre til tests, så er det da helt klart relevant. Vidste ikke, at den også kunne det.
Men ja, det kan være, det skal være et PR for sig :)